### DISCUSSION TITLE
Set voice channel status?

### DISCUSSION THREAD

**Discussion by ClaytonAstrom** [serenity-rs/serenity#2758] [Q&A] [answered]
Is there a way to programmatically change the voice channel status?

**Comment by jamesbt365** [comment, marked as answer]
I implemented the gateway events and audit log stuff for this, but I couldn't implement changing the voice channel status at the time because there was a lock on bots doing it, now that has been removed, it can be implemented.

Its up for grabs if nobody implements it before I do.
